### PR TITLE
Feature/zip 179 create cron job to charge clients each week

### PR DESCRIPTION
**Pull request by aalishanjami**

### CODE REVIEW

File: components/unzipped/dashboard/DashboardTable.js [hunk 1]
```diff
@@ -44,34 +36,48 @@ const Box = styled.div`
...
-        }, (time || 500));
+    const generatePopout = (userType, item) => {
+        if (userType === 0 || userType === 2) {
+            return [
+                {
+                    text: 'Invoice',
+                    onClick: () => router.push(`projects/client/invoice/${item._id}`),
+                },
+                {
+                    text: 'Details',
```
Comment (zubairaltaf10) [inline components/unzipped/dashboard/DashboardTable.js:53]: Can you please elaborate about these logs? if its not necessary then remove it

File: pages/dashboard/projects/client/invoice/[id].js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+const FounderInvoice = ({ projectName, invoiceTaskHours, access_token, getBusinessTasksByFounder }) => {
+    const router = useRouter();
+    const { id } = router.query;
+
+    const [data, setData] = useState([]);
+    const [filteredData, setFilteredData] = useState([]);
+    const [sortedData, setSortedData] = useState({})
+    const [selectedWeek, setSelectedWeek] = useState(null);
+    const [weekOptions, setWeekOptions] = useState([]);
+    const [take, setTake] = useState(25);
```
Comment (zubairaltaf10) [inline pages/dashboard/projects/client/invoice/[id].js:30]: is there any option for user to select number of invoices to be shown on gird like a dropdown 10,15,20,25,50,100 like that or is it fixed to 25?

File: components/unzipped/dashboard/DashboardTable.js (hunk 1 above)
Comment (aalishanjami) [reply components/unzipped/dashboard/DashboardTable.js:53]: These are already there.

File: pages/dashboard/projects/client/invoice/[id].js (hunk 2 above)
Comment (aalishanjami) [reply pages/dashboard/projects/client/invoice/[id].js:30]: No, it vary. There are three options on top 25 50 100.

**Pull request merged**